pkg/build/builder/daemonless.go

```diff
@@ -555,6 +561,9 @@ func GetDaemonlessClient(systemContext types.SystemContext, store storage.Store,
 		return nil, fmt.Errorf("unrecognized BUILD_ISOLATION setting %q", strings.ToLower(isolationSpec))
 	}
 
+	isolation = buildah.IsolationOCI
```

Comment:
Nalin did this for convenience but you can also force this by setting the `BUILD_ISOLATION` env var on the build